ci/container_scripts/install_deps.sh

```diff
@@ -18,6 +18,7 @@ APT_PACKAGES="
   python3
   python3-pip
   xz-utils
+  rsync
```

Comment:
These dep lists are useful as the canonical list of deps needed for Shadow. Let's add a comment here that this *isn't* needed for Shadow in general; only for our CI infrastructure.

Reply:
I put these under different variables `APT_CI_PACKAGES` and `RPM_CI_PACKAGES` to make this clearer.